src/components/homepage/main-components/SignUp.tsx

```diff
@@ -128,7 +128,7 @@ const SignUp = () => {
             type="email" 
             name="" 
             id="EMAIL" 
-            autoFocus
+            // autoFocus
```

Comment:
Remove it fully instead of commenting it out.

Reply:
Solved in #147